admin/data/sql/migrations/20200710143820.undo.check-retro-input-assistant.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE [mtc_admin].[check] DROP COLUMN retroInputAssistantFirstName
```

Comment:
These columns can be (more efficiently?) dropped in a single statement:
https://github.com/jon-shipley/patterns/blob/master/mssql-idempotent-sql.md#Columns